snuba/replacer.py

```diff
@@ -30,6 +30,90 @@
 NODES_REFRESH_PERIOD = 10
 
 
+class ShardedConnectionPool(ABC):
```

Comment:
Why did you add this base class? There is only one possible subclass.

Reply:
I am thinking about a round robin load balancer that ignore the write nodes, but I did not implement it yet. That would make two implementations. Also having the base class will make it more reusable as we move the load balancer back into the clickhouse access package.

Reply:
Got it. Thanks.